main.js

```diff
@@ -68,6 +79,69 @@ function main(args) {
     }
 }
 
+function createStatsClient(val) {
+    var opts = null;
+    var matchesHostPort = val.match(/^(.+):(\d+)$/);
+    if (matchesHostPort) {
+        opts = {
+            host: matchesHostPort[1],
+            port: parseInt(matchesHostPort[2])
+        };
+    } else {
+        var file = path.resolve(val);
+        opts = {
+            _ephemeralSocket: new FileStatsLogger(file)
+        };
+    }
+
+    var createStatsdClient;
+
+    try {
```

Comment:
By using a `try/catch` here we're able to use `uber-statsd-client` without adding it as a dependency.